randovania/games/dread/json_data/Cataris.txt

```diff
@@ -2662,7 +2669,7 @@ Extra - asset_id: collision_camera_031
   * Extra - right_shield_def: None
   > Dock to EMMI Zone Tower East
       Any of the following:
-          Space Jump or Infinite Bomb Jump (Beginner)
+          Space Jump or Speed Booster or Infinite Bomb Jump (Beginner)
```

Comment:
Make this Simple IBJ rather than the trick. This can cause you to go this way logically while being unable to IBJ